classes/report/fields/runner/coverage/html.php

```diff
@@ -75,7 +75,7 @@ public function __toString()
 
                 ksort($classes, \SORT_STRING);
 
-                foreach ($classes as $className => $classFile) {
+                foreach (array_keys($classes) as $className) {
```

Comment:
Same comments, `array_keys` allocates a new array. Either we don't care about the values of `$classes` and then we should think about another structure, or use the `$className => $_` binding.